contrib/scripts/kind.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
```

Comment:
Please add the copyright and SPDX headers.

Have you also considered setting the `-e` option for bash to make it exit immediately if an error is encountered?

Reply:
oh right, I removed debug flag and by mistake removed -e as well. Thanks :100: